backend/task.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+from sklearn.cluster import KMeans
+import pandas as pd
+import numpy as np
+import re
+import nltk
+import multiprocessing
+from gensim.models import Word2Vec
+from time import time  # To time our operations
+from collections import defaultdict  # For word frequency
+import logging  # Setting up the loggings to monitor gensim
+logging.basicConfig(format="%(levelname)s - %(asctime)s: %(message)s", datefmt= '%H:%M:%S', level=logging.INFO)
+
+import matplotlib.pyplot as plt
+
+data = pd.read_csv("floop_data_15k.csv")
+
+data.shape
+
+data.head()
+
+data.info()
```

Comment:
why do we need to read the head of the table and look at the general info every time the script executes?

Reply:
I have deleted both `data.head()` and `data.info()`